Xamarin.Forms.Platform.WPF/CellControl.cs

```diff
@@ -93,11 +93,11 @@ protected override void OnRenderSizeChanged(SizeChangedInfo sizeInfo)
 				{
 					foreach (var child in vc.LogicalChildren)
 					{
-						if (child is Layout layout)
+						if (child is VisualElement visualElement)
```

Comment:
This allow us to stretch the view. It is implemented to be consistent with other platforms